wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/SchedulingRecursionTest.java

```diff
@@ -52,9 +57,9 @@ public void initialize() {
     }
   }
 
-  @ValueSource(booleans = {true, false})
+  @EnumSource(InterruptionBehavior.class)
   @ParameterizedTest
-  void defaultCommand(boolean interruptible) {
+  void defaultCommand(InterruptionBehavior interruptionBehavior) {
```

Comment:
the name of this test seems incomplete

Reply:
Do we want to verify that default commands are interruptible? The old impl didn't give an API option for otherwise, but now that it is possible I'm wondering.

Reply:
it wouldn't hurt